baseframe/__init__.py

```diff
@@ -196,6 +196,8 @@ def init_app(self, app, requires=[], ext_requires=[], bundle_js=None, bundle_css
 
         app.config['tz'] = timezone(app.config.get('TIMEZONE', 'UTC'))
 
+        app.config['theme'] = theme
```

Comment:
Let's validate the theme before doing this, by having a list of recognised theme names at the top of this file. Basically:

```python
THEME_FILES = {
    'bootstrap3': {'autoform.html.jinja2': 'baseframe/bootstrap3/autoform.html.jinja2', …},
    'mui': {'autoform.html.jinja2': 'baseframe/mui/autoform.html.jinja2', …}
    }
…
if theme not in THEME_FILES:
    raise ValueError("Unrecognised theme: %s" % theme)
```

Reply:
Fixed